### PR TITLE
added russian localization

### PR DESCRIPTION
**Pull request by arirays**
# What it Does
* Adds Russian localization
* Discussion in #7 

# How I Tested
* Set system language to Russian
* Start app

### CODE REVIEW

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -878,6 +879,7 @@
 				"LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]" = "@executable_path/../Frameworks";
 				MACOSX_DEPLOYMENT_TARGET = 13.4;
 				OTHER_LDFLAGS = "-ObjC";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.aris.Basic-Car-Maintenance";
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/project.pbxproj]: Note that you should _not_ have changed the bundle ID, look at the steps in the [README](https://github.com/mikaelacaron/Basic-Car-Maintenance#getting-started) the bundle ID you created should be in the `xcconfig` file, and NOT be changed in the project

Comment (mikaelacaron) [reply Basic-Car-Maintenance.xcodeproj/project.pbxproj]: I'm going to manually change this, and fix it, but note this for the future contributing to projects

**Review by mikaelacaron — APPROVED** [review]
Thank you for contributing! Note that you shouldn't have changed the bundle ID, that was noted in the README on how to set up the project with the xcconfig file.

I'm going to manually fix it for you, and merge this

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/project.pbxproj]:
```suggestion
```

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -918,6 +920,7 @@
...
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/project.pbxproj]:
```suggestion
```

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1023,6 +1026,7 @@
 					"@executable_path/Frameworks",
 					"@executable_path/../../Frameworks",
 				);
+				PRODUCT_BUNDLE_IDENTIFIER = "com.aris.Basic-Car-Maintenance";
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/project.pbxproj]:
```suggestion
```

File: Basic-Car-Maintenance.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1050,6 +1054,7 @@
...
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/project.pbxproj]:
```suggestion
```

**Comment by arirays** [comment]
Thank you @mikaelacaron ! 👍

**Pull request merged**